### DISCUSSION TITLE
VMware and placeholder VM (need exclude)

### DISCUSSION THREAD

**Discussion by netbja** [bb-Ricardo/netbox-sync#130] [Ideas]
Hello to all,

I use NetBox-Sync and it's really well done. However, I had a small request at the end of the year.
Between two sites A and B, I have a VMware replication with Site Recovery Manager (SRM). 
I don't want to synchronize in NetBox the replicated VMs aka placeholder virtual machine of site B. 
But take only the running VMs on B site.
Here is an example that I think is good to help developers:

https://communities.vmware.com/t5/VMware-PowerCLI-Discussions/PowerCLI-How-to-distinguish-real-vm-from-SRM-placeholder/m-p/2214006

Thanks.
Bernard.

**Comment by bb-Ricardo** [comment]
Hi,

Thank you for this info. Wasn't aware of SRM but it seems quite easy to implement and add a config var for this to optionally disable the symc of placeholder VMs.

**Reply by netbja** [reply]
Hi,
Perfect, I'm standing by to test this...
Cheers.

**Reply by bb-Ricardo** [reply]
Hi Bernard,

I added this option to current 'development' branch. Can you test it and give me some feedback? Please use issue #136 for that
